examples/with-apollo/components/PostList.js

```diff
@@ -93,34 +19,96 @@ export const allPosts = gql`
 `
 export const allPostsQueryVars = {
   skip: 0,
-  first: POSTS_PER_PAGE
+  first: 10
 }
 
-// The `graphql` wrapper executes a GraphQL query and makes the results
-// available on the `data` prop of the wrapped component (PostList)
-export default graphql(allPosts, {
-  options: {
-    variables: allPostsQueryVars
-  },
-  props: ({ data }) => {
-    return ({
-      data,
-      loadMorePosts: () => {
-        return data.fetchMore({
-          variables: {
-            skip: data.allPosts.length
-          },
-          updateQuery: (previousResult, { fetchMoreResult }) => {
-            if (!fetchMoreResult) {
-              return previousResult
+export default function PostList () {
+  return (
+    <Query query={allPostsQuery} variables={allPostsQueryVars}>
+      {({ loading, error, data: { allPosts, _allPostsMeta }, fetchMore }) => {
+        if (error) return <ErrorMessage message='Error loading posts.' />
+        if (loading) return <div>Loading</div>
+
+        function loadMorePosts () {
```

Comment:
Can we move this function declaration outside of and below `PostList`? Everything else looks awesome to me. Thank you!!

Reply:
@adamsoffer Sure thing! I'm guessing you'd like the same done in the modules PostUpvoter (upvotePost function) and Submit (handleSubmit function)?

Reply:
Ah yes those as well thank you!

Reply:
@adamsoffer done 👍